src/components/ArtistName.jsx

```diff
@@ -0,0 +1,8 @@
+
+
+export const ArtistName = () => {
```

Comment:
A map method will be used for mapping through different artists and because the json is using an array, the  index will be important to pass on as a 2nd  property. 
E.g: `export const ArtistName = ({ artistslist }) => {
    return (
        <div>
            {artistslist.map((artist, index) => (
                <div key={artist.id} >
                    <a
                        href={artist.external_urls.spotify}
                        target="_blank"
                        rel="noopener noreferrer"
                       >
                       {artist.name}
                     </a>                  
                </div>
            ))}
        </div>
    )
}`